README.md

```diff
@@ -1,23 +1,29 @@
----
-# Evaluator Samples
+# Coursemology Evaluator
```

Comment:
We already have another repository for the actual evaluator (https://github.com/coursemology/evaluator-slave) so this title is a bit confusing.